### ISSUE TITLE
Can we run generated structures through CheckCIF in a programmatic way?

### ISSUE THREAD

**Issue by sgbaird** [sparks-baird/xtal2png#63]
https://checkcif.iucr.org/

E.g. if we wanted to CheckCIF 10 million structures, is the "Send CIF for checking" button through the web app the only option?

This may also give hints to certain types of post-processing steps that we might undertake, though many of these may be less necessary if we use a reasonable symprec (?) tolerance when writing the CIF files to disk.

**Comment by kjappelbaum** [comment]
Not sure what you're aiming for here, but [`cod-tools`](https://wiki.crystallography.net/cod-tools/) implements a subset of checkcif in `cif_cod_check`.

If you just want to check for atomic overlaps and "sanity" there is not that much good stuff out there for crystal except for a few hand-crafted rules

**Comment by sgbaird** [comment]
@kjappelbaum thank you! This is great info.

**Comment by sgbaird** [comment]
Looks like it's also possible to have a local installation, at least based on what's described in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6944088/

**Comment by kjappelbaum** [comment]
yep, indeed [Platon](http://www.platonsoft.nl/platon/pl000601.html) gives this option
